roombasim/pittras/config.py

```diff
@@ -75,3 +75,6 @@
 
 # Tolerance for distance comparasion in the HoldPositionTask
 PITTRAS_HOLD_POSITION_TOLERANCE = 0.2
+
+# Tolerance for speed comparasion in the VelocityTask
+PITTRAS_VELOCITY_TOLERANCE = 0.2
```

Comment:
This seems high, maybe more like `0.05` or `0.1`?

Reply:
Sure :)